UPchieve/web
# force the iframe to make requests until google document is loaded

Description
-----------
- Request for the embedded Google document every 2 seconds until it is properly loaded to the DOM. This fixes an issue where the Google document iframe for UPchieve 101 may respond with a status code of `204` and not load on to the DOM since there was no content received
- Added the loader component while the google document is being fetched

Steps to test:
1. Visit the `/training` as a volunteer
2. Select the dropdown `Training Courses`
3. Either click `Start Course` or `Review Course` for UPchieve 101
Note: There are several modules/dropdowns that render a Google Doc and sometimes these do not show when the response status code is 204. If given a 204 response, this PR retries the request for the doc to embed in the iframe until we get a 200 status code response. I used the Network tab `Doc` to check the requests
4. Click on `Becoming an Active Coach` > `Guide to choosing your references` (or any other set of dropdowns that would render a Google Doc)
5. Monitor the Network Requests under `Doc` as you open and close the dropdown with the Google Doc. You may need to refresh a few times and repeat `Step 4` to get a response that is a 204 status code
6. When given a status code of 204 you should see additional requests sent out every 2 seconds for that iframe until it receives a status code of 200


Developer self-review checklist
-------------------------------
